### ISSUE TITLE
Merge parts of https://gitlab.com/macrominds/provision/uberspace/readme

### ISSUE THREAD

**Issue by tpraxl** [friesenkiwi/ansible-collection-uberspace#7]
I very much appreciate what you accomplished with your ansible collection.
I find your work very inspiring and useful.

If you are interested in merging useful parts of my own "collection" over at https://gitlab.com/macrominds/provision/uberspace/readme , I'd offer my support. 

I think other Ubernauts out there could benefit from that, and I would be happy to collaborate.

**Comment by friesenkiwi** [comment]
Yep, definitely interested to merge! I also had a look at your work and had the following idea:

* Manage [mail domains](https://gitlab.com/macrominds/provision/uberspace/mail-domains) => Not yet covered, but easily includable into https://github.com/friesenkiwi/ansible-collection-uberspace/blob/master/roles/uberspace_account/tasks/domains.yml#L10 or as `_mail`
* Manage [mail accounts](https://gitlab.com/macrominds/provision/uberspace/mail-accounts) => new, let's include as `_mail`
* Setup a folder based [mail autoresponder](https://gitlab.com/macrominds/provision/uberspace/mail-autoresponder) => new, let's include as `_mail`
* [Install lib webp](https://gitlab.com/macrominds/provision/uberspace/lib-webp) to convert your images to *.webp for improved compression => quite specific, maybe as `_webhosting`?
* Manage [web domains](https://gitlab.com/macrominds/provision/uberspace/web-domains) => covered by https://github.com/friesenkiwi/ansible-collection-uberspace/blob/master/roles/uberspace_account/tasks/domains.yml#L10
* Put a [private deployment key](https://gitlab.com/macrominds/provision/uberspace/deployment-key) onto your server to be able to checkout private git repos => unsure, seems to be a bit specific or even general :D maybe better generate they key and download the public part? (e.g. https://docs.ansible.com/ansible/2.9_ja/modules/openssh_keypair_module.html / https://github.com/friesenkiwi/ansible-collection-uberspace/blob/master/roles/uberspace_account/tasks/key.yml#L14)
* Manage the [php version](https://gitlab.com/macrominds/provision/uberspace/web-prerequisites) => covered by https://github.com/friesenkiwi/ansible-collection-uberspace/blob/master/roles/uberspace_account/tasks/tools.yml#L8
* Turn the Uberspace [html folder into a symlink](https://gitlab.com/macrominds/provision/uberspace/web-link-html-to-deployer-target) to your deployment target => new, let's include as` _webhosting`
* Setup a [shared .env](https://gitlab.com/macrominds/provision/uberspace/web-provide-shared-deployer-dot-env) for deployer => unsure?
* Manage [log rotate](https://gitlab.com/macrominds/provision/uberspace/web-logrotate) => new, let's include as `_webhosting`

Generally, merging things into one role/collection and using `include_role`/`import_role` - `tasks_from` might reduce boilerplate.
